src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.php

```diff
@@ -37,9 +36,6 @@
             ->public()
             ->tag('cache.pool', ['clearer' => 'cache.app_clearer'])
 
-        ->set('cache.app.taggable', TagAwareAdapter::class)
```

Comment:
I'd prefer keeping this definition as is

Reply:
And just change to an alias if applicable in the extension?

Reply:
Yes